src/components/ExtraPages/Help/IndicesHelp.js

```diff
@@ -1,95 +1,787 @@
+                  {" "}
+                  <HashLink
+                    to="/indices-help#extreme"
+                    scroll={(el) =>
+                      el.scrollIntoView({ behavior: "auto", block: "center" })
+                    }>
+                    Extreme Events
+                  </HashLink>
+                </li>
+                <li>
+                  {" "}
+                  <HashLink
+                    to="/indices-help#impact"
+                    scroll={(el) =>
+                      el.scrollIntoView({ behavior: "auto", block: "center" })
+                    }>
+                    Impact Related
+                  </HashLink>
+                </li>
+                <li>
+                  {" "}
+                  <HashLink
+                    to="/indices-help#degree"
+                    scroll={(el) =>
+                      el.scrollIntoView({ behavior: "auto", block: "center" })
+                    }>
+                    Degree Days and Seasons
+                  </HashLink>
+                </li>
+              </ul>
+            </Nav>{" "}
+          </Container>
         </Col>
+        {/* page contents */}
+        <Col className="ml-5">
+          <div>
+            <h1>Information about Indices</h1>
+            <br></br>
+            <h3>How the Indices are Calculated</h3>
+            <p>
+              Unless stated otherwise, a seasonal time series of each index is
+              first computed from daily model output on the native grid, then
+              averaged over each time period to obtain a set of seasonal model
+              climatologies. These seasonal climatologies are then regridded
+              onto a common 12km grid (OSGB) using area weighting, so that
+              output from models on different grids can be compared directly.
+              For regional climate model output (EuroCORDEX and UKCP18 12km
+              runs), a land-sea mask is applied to remove any so that only cells
+              with 1 more than 50% land coverage are included; values in these
+              land surface cells are regridded onto the UK land surface grid.
+              For global climate model output (CMIP5 and UKCP18 60km runs), the
+              low-resolution data is regridded directly onto the land surface
+              12km grid. No bias correction has been carried out. To facilitate
+              comparison between models that use different calendars, any
+              indices that depend on the number of days in the season or year
+              (for example, number of tropical nights tr or accumulated
+              precipitation prcptot) are standardised by averaging over the
+              number of days in the model output and multiplying by 90 (for
+              seasonal indices) or 365 (for annual indices).
+            </p>
+          </div>
+          <br></br>
+          <h6>
+            The following describes the indices for which plots are available.
```

Comment:
```suggestion
            This page describes the indices for which plots are available. The process used for index calculation is described in the overview section of these help pages.
```

Reply:
'overview section' needs to link to the relevant page

Reply:
Will wait and see if this really needs to be a separate page or if a separate link to the indices page will suffice